client/src/app.js

```diff
@@ -109,6 +109,26 @@
 
     });
 
+    navigator.geolocation.getCurrentPosition(success, error, options);
+
+    function success(position) {
+      var lat = position.coords.latitude;
+      var long = position.coords.longitude;
+      heroFactory.setOrder({ currentLocation: [lat, long] });
+      requesterFactory.setOrder({ currentLocation: [lat, long] });
+    }
+
+    var options = {

```

Comment:
looks like your calling a function and passing in "options" before you declare it as a variable